collection-scripts/gather

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Resource List
+resources=()
+
+# Operator Resources
+resources+=(clusteroperators)
+
+# Certificate Resources
+resources+=(certificatesigningrequests)
+
+# Machine/Node Resources
+resources+=(nodes machines machineconfigs)
+
+# Namespaces/Project Resources
+resources+=(ns/default ns/openshift ns/kube-system)
```

Comment:
why is `ns/openshift` interesting?

Reply:
To pull content from the OpenShift Namespace? 
I'm not sure what this question is trying to ask.